developer-docs-site/docs/move/move-on-aptos/cryptography.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Cryptography"
+slug: "cryptography"
+---
+
+# Cryptography in Move
+
+The Move language originally supported only four basic cryptographic operations:
```

Comment:
maybe start with a high level of the cryptographic functions we introduce into move followed by the principles

Reply:
Fixed.